src/components/question_creation/question-creation-page.tsx

```
@@ -0,0 +1,160 @@
+"use client";
+
+import React from "react";
+import TopBar from "./top-bar";
+import QuestionTypeSelector, { QuestionType } from "./question-type-selector";
+import QuestionEditor, { QuestionData } from "./question-editor";
+import QuestionSettings from "./question-settings";
+import ValidationErrorModal from "./validation-error-modal";
+import { validateQuestionData, ValidationError } from "./validation";
+
+interface QuestionSettings {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'QuestionSettings'. Consider to delete it or rename it.

'QuestionSettings' is defined here:


(lint/suspicious/noRedeclare)